Explain how to use Terramate with and without Git integration (#1007)
terramate-io/terramate: docs/getting-started/index.md

> If you haven’t installed Terramate yet, please refer to the [installation](../installation.md) section for various options.

To start using Terramate, there are two possible ways of configuring a Terramate project. If you invoke Terramate inside
a Git repository, Terramate will automatically assume the top-level of your repository as the project root.

> Suppose you use Terramate in a normal directory that isn't a Git repository. In that case, you must configure the project

Reviewer: ```suggestion
If you want to use Terramate in a directory that isn't a git repository, you must configure the project
```

Author: accepted - will still write **Git** though as we do it similar in all other pages of the documentation
